framework/include/media/MediaRecorder.h

```diff
@@ -191,7 +191,7 @@ class MediaRecorder
 
 	/**
 	 * @brief Set limitation of recording time by given value(second), will be stopped when it reaches that.
-	 * This should be called after create but before prepare
```

Comment:
In mediarecorder_main.cpp, it seems that setDuration(line 161) is called after setDataSource(line 150, 154, and 158).
Could you check again this change is correct?

Reply:
yes, it is correct.
create --> setDataSource -> setDuration or not -> prepare

Reply:
what is problem~?

Reply:
If my first comment is correct, the description needs to be changed as "before setDataSource -> after setDataSource".

Reply:
ah, yes i got it.
@chanijjani very thanks!